hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsUserAggregateFactory.java

```diff
@@ -28,7 +28,7 @@ private MetricsUserAggregateFactory() {
   }
 
   public static final String METRIC_USER_ENABLED_CONF = "hbase.regionserver.user.metrics.enabled";
-  public static final boolean DEFAULT_METRIC_USER_ENABLED_CONF = true;
+  public static final boolean DEFAULT_METRIC_USER_ENABLED_CONF = false;
```

Comment:
We have many default constants for multiple configs defined in core code, seems like we are not utilizing hbase-default.xml.
@saintstack it's not related to this PR, but should we have a general guideline to keep config's default values only in `hbase-default.xml` and not in code?

Reply:
Sounds find if the original config is in hbase-default; this one is not.

We don't have all configs in hbase-default. Too many. Operator would get lost. The boundary between listing in hbase-default or not is not specified unfortunately.  A task to do.